bom/pom.xml

```diff
@@ -253,6 +253,31 @@
         <version>4.1.68.Final-SNAPSHOT</version>
         <classifier>osx-x86_64</classifier>
       </dependency>
+
+      <!-- Add netty-tcnative* as well as users need to ensure they use the correct version -->
+      <dependency>
+        <groupId>io.netty</groupId>
+        <artifactId>netty-tcnative</artifactId>
+        <version>${tcnative.version}</version>
+        <classifier>linux-x86_64</classifier>
+      </dependency>
+      <dependency>
+        <groupId>io.netty</groupId>
+        <artifactId>netty-tcnative</artifactId>
+        <version>${tcnative.version}</version>
+        <classifier>linux-aarch_64</classifier>
+      </dependency>
+      <dependency>
+        <groupId>io.netty</groupId>
+        <artifactId>netty-tcnative</artifactId>
```

Comment:
Perhaps `${tcnative.artifactId}` for consistency.

Reply:
No we want to define these explicit